tools/FakeItEasy.PrepareRelease/FakeItEasy.PrepareRelease.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Octokit" Version="0.26.0" />
+  </ItemGroup>
+
+</Project>

```

Comment:
Note: I built this project but didn't actually try to run it, since there's no "dry run" option.

Reply:
I usually just modify it to return early before we start modifying things. Right after the new check for release notes/milestone agreement.

Reply:
👍 
I'll do that.

Reply:
Well, good thing I tried, because it was broken. The .githubtoken is not in the same directory, but in the parent directory. We could keep the code as is and just move the file, but if we make other tools that use it, it would be better to have it in a common location. What would you prefer?

Reply:
For now, I just manually moved the file; you'll have to do the same. I can change it if you like.

Reply:
Yes